scripts/deployment/deploy.sh

```diff
@@ -150,6 +150,16 @@ case $opt in
             docker-compose -f docker-compose-${env}.yml up -d statsd-exporter
             echo "Completed deploy operation."
             ;;
+        deploy-node-exporter)
```

Comment:
@burnerlee if push gateway and node exporter is part of nginx config it needs to be deployed with EvalAI backend and frontend. Please make the changes in `auto_deploy` command to start node exporter and push gateway. See line 50 for reference